src/DIRAC/Core/Security/Properties.py

```diff
@@ -74,3 +74,7 @@
 
 #: Allow staging files
 STAGE_ALLOWED = "StageAllowed"
+
+#: Allow VMDIRAC Operations via various handlers
+VM_WEB_OPERATION = "VmWebOperation"
```

Comment:
We can probably replaced these properties by Operator in order not to multiply entities for both these properties
```suggestion
VM_WEB_OPERATION = "Operator"
```

Reply:
I've replaced VmWebOperation with Operator. I've left the RPC one as that's generally only assigned to the hostcert used for the VMs, so doesn't really fit any of the existing ones.